src/vos/vos_iterator.c

```diff
@@ -534,6 +535,7 @@ vos_iterate(vos_iter_param_t *param, vos_iter_type_t type, bool recursive,
 
 	anchor = type2anchor(type, anchors);
 
+	vos_dtx_tbc_hold(tbc);
```

Comment:
Is there a reason this is inside of vos_iterate rather than just in the verification call ?   I see you pass NULL everywhere except one place

Reply:
The @tbc is used by VOS to record the DTXs that should be committed during the availability check for the iteration. The verification logic is on client side, it cannot (and should not) directly talk with related DTX logic inside VOS. So it is the server side enumeration RPC handler (that serves for verification logic) to handle the @tbc logic.